application/src/main/java/org/togetherjava/tjbot/commands/tag/TagManageCommand.java

```diff
@@ -196,6 +189,8 @@ private void createIdTag(SlashCommandEvent event) {
             if (failure instanceof ErrorResponseException ex
                     && ex.getErrorResponse() == ErrorResponse.UNKNOWN_MESSAGE) {
                 event.reply("This message doesn't exist").setEphemeral(true).queue();
+            } else {
+                failure.printStackTrace();
```

Comment:
(see above about rethrow and logger)

Reply:
Done by printing the exception to the logger